src/multisig/multisig.cpp

```diff
@@ -33,19 +33,21 @@
 #include "cryptonote_basic/account.h"
 #include "cryptonote_basic/cryptonote_format_utils.h"
 #include "multisig.h"
+#include "cryptonote_config.h"
 
 #undef MONERO_DEFAULT_LOG_CATEGORY
 #define MONERO_DEFAULT_LOG_CATEGORY "multisig"
 
 using namespace std;
 
-static const rct::key multisig_salt = { {'M', 'u', 'l', 't' , 'i', 's', 'i', 'g', 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 } };
-
 namespace cryptonote
 {
   //-----------------------------------------------------------------
   crypto::secret_key get_multisig_blinded_secret_key(const crypto::secret_key &key)
   {
+    rct::key multisig_salt;
+    memcpy(multisig_salt.bytes, config::HASH_KEY_MULTISIG, sizeof(rct::key));
```

Comment:
Although the config value happens to match `sizeof(rct::key)`, this seems a bit confusing for future readers. I presume the config value is declared an unsigned char array because the `rct::key` type isn't available, so I would suggest a static assertion here that `sizeof(rct::key) == sizeof(config::HASH_KEY_MULTISIG)` in order to clarify the intent and reduce the potential for future accidents (a stray delete in the config could end up copying from uninitialized memory, which may or may not produce a test failure).

Reply:
Good point. Will update.

Reply:
Would be better to use compile-time `static_assert` (that @iamamyth suggested) instead of runtime `CHECK_AND_ASSERT_THROW_MES` check.

Reply:
Done.